src/stdlib/lv_mem.c

```diff
@@ -116,6 +116,17 @@ void * lv_malloc_zeroed(size_t size)
     return alloc;
 }
 
+void * lv_calloc(size_t num, size_t size)
```

Comment:
Do we need `calloc` too? For me
```c
calloc(12, 30);
zalloc(12 * 30);
```

are very similar, so I wonder why we need a new function to use `(..., ...)` instead of `(... * ...)`